### PR TITLE
feat(proxy): respect https_proxy env var

### PR DESCRIPTION
**Pull request by wyvern8**
### All Submissions:

* [ yes] Have you followed the guidelines in our Contributing document?
* [ yes] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submissions:

1. [ no] Have you created new unit?
2. [ yes] Does your submission pass all tests?
3. [ yes] Have you lint your code locally prior to submission (automated)?

### CODE REVIEW

**Comment by wyvern8** [comment]
another pr for review @Neko-Design .  i just used local squid for testing - we may need to add tls bump support later.

File: src/LaunchDarklyApiClient.js [hunk 1]
```diff
@@ -20,6 +22,13 @@ export class LaunchDarklyApiClient {
     static async create(API_TOKEN, log, swaggerYamlString) {
         log.debug(`creating api client with token: ${API_TOKEN}`);
 
+        let proxy = process.env.https_proxy;
```
Comment (Neko-Design) [inline src/LaunchDarklyApiClient.js]: Possibly check for http_proxy/https_proxy as depending on the system either could be set?

**Review by Neko-Design — APPROVED** [review]
Looks good, just need to consider which proxy variations we'll look at, all caps etc.

**Pull request merged**